pkg/products/cloudresources/reconciler.go

```diff
@@ -439,6 +446,12 @@ func (r *Reconciler) cleanupResources(ctx context.Context, installation *integre
 }
 
 func (r *Reconciler) reconcileSubscription(ctx context.Context, serverClient k8sclient.Client, inst *integreatlyv1alpha1.RHMI, productNamespace string, operatorNamespace string) (integreatlyv1alpha1.StatusPhase, error) {
+
+	_, err := r.waitAddonCidrPresentInStrategy(ctx, serverClient)
```

Comment:
I think we can maybe do this using the existing code to set the CIDR block - this code is mostly duplicated from there and the boolean return is unused